SingularityService/src/main/java/com/hubspot/singularity/data/DisasterManager.java

```diff
@@ -68,20 +68,21 @@ public SingularityDisabledAction getDisabledAction(SingularityAction action) {
       new SingularityDisabledAction(
         action,
         String.format(MESSAGE_FORMAT, action, DEFAULT_MESSAGE),
-        Optional.<String>empty(),
+        Optional.empty(),
```

Comment:
Clarification for me, why are you removing the <String> and <Long>?

Reply:
general cleanup. They were needed in older java versions, but not in newer ones. Been cleaning up files as I happen to edit them for other reasons